app/src/main/java/com/globant/harrypotterapp/fragment/CharactersFragment.kt

```diff
@@ -76,6 +79,7 @@ class CharactersFragment : Fragment(), OnCharacterClicked {
     }
 
     companion object {
+        private const val CHARACTERS = "Characters"
```

Comment:
why did you need this? you are only using it in the line 68 in a getString. put this in the same resource

Reply:
Refactored to string resource.